### DISCUSSION TITLE
Sharing some feature usage statistics

### DISCUSSION THREAD

**Discussion by lavinir** [lavinir/hassio-onedrive-backup#170] [Announcements]
Hi all,

Just thought I'd share some telemetry from the users who have opted to enable anonymous telemetry. Currently I estimate only about 2% of addon users have this enabled and I do encourage you to turn this on so I have more visibility which features are more used and focus on them.

Here are the current statistics from the users sharing telemetry:
![image](https://github.com/lavinir/hassio-onedrive-backup/assets/56975945/cc2967ec-7710-4fb7-bcd7-a21ca91788f0)

If you want more details to exactly what is shared in the anonymous telemetry, this is documented [here](https://github.com/lavinir/hassio-onedrive-backup?tab=readme-ov-file#enable-anonymous-telemetry)

**Comment by donverse** [comment]
Thanks; turning this on now for your further information. Keep up the great work!
